framework/src/media/MediaWorker.cpp

```diff
@@ -37,8 +37,12 @@ void MediaWorker::startWorker()
 	++mRefCnt;
 	medvdbg("MediaWorker::startWorker() - increase RefCnt : %d\n", mRefCnt);
 	if (mRefCnt == 1) {
+		pthread_attr_t attr;
+		pthread_attr_init(&attr);
+		pthread_attr_setstacksize(&attr, mStacksize);
 		mIsRunning = true;
-		mWorkerThread = std::thread(std::bind(&MediaWorker::mediaLooper, this));
+		pthread_create(&mWorkerThread, &attr, (pthread_startroutine_t)&MediaWorker::mediaLooper, this);
+		pthread_setname_np(mWorkerThread, mThreadName.c_str());
```

Comment:
Don't we need to check the return type of pthread_create before attempting to set a name for it ?  Because there are several scenarios in which pthread_create would fail